fli-iam/shanoir-ng
# shanoir-issue#2446: add list of studies of subject on subjectDelete

How to test:

Open subject-list
Click on the delete button
The popup should display the studies in which the subject belong:
![image](https://github.com/user-attachments/assets/83490e65-ecee-48d2-a682-0d9e16949c2c)

Do the same but not from the subject-list but from a view with a tree:
![image](https://github.com/user-attachments/assets/ed350e37-34ee-4ae4-8e23-94bbc5a30039)
